go-controller/pkg/clustermanager/egressip_controller.go

```diff
@@ -1042,6 +1042,58 @@ func (eIPC *egressIPClusterController) reconcileEgressIP(old, new *egressipv1.Eg
 	return nil
 }
 
+func (eIPC *egressIPClusterController) syncEgressIPs(ips []interface{}) error {
+	if !util.PlatformTypeIsEgressIPCloudProvider() {
+		return nil
+	}
+	cloudPrivateIPConfigMap, err := eIPC.getCloudPrivateIPConfigMap()
+	if err != nil {
+		return fmt.Errorf("syncEgressIPs unable to get cloud private ip config: %w", err)
+	}
+	egressIPs, err := eIPC.kube.GetEgressIPs()
+	if err != nil {
+		return fmt.Errorf("syncEgressIPs unable to get Egress IPs: %w", err)
+	}
+	for _, egressIP := range egressIPs.Items {
+		updatedStatus := []egressipv1.EgressIPStatusItem{}
+		ipNotFound := false
```

Comment:
`ipNotfound` varable is too generic -> its not conveying the logic. To me it means "egressIP not found in cloudPrivateIPConfigMap cache" but reality is "egressIP not found in cloudPrivateIPConfigMap cache and the status nodeName found on EIP is not the same as we have on cloudPrivateIPConfig" :) -> you see my point? I think this variable naming has to be changed to reflect what it stands for?